### ISSUE TITLE
Abnormal circulating Fetuin A concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5996]
**Preferred term label:** Abnormal circulating Fetuin A concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of fetuin A in the blood is outside the normal range., PMID: 22074587

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:**  Decreased circulating Fetuin A concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of fetuin A in the blood is below the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Fetuin A concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:**  Increased circulating Fetuin A concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of fetuin A in the is blood above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Fetuin A concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
adding terms.
